test/guessTheNumber_test.js

```diff
@@ -0,0 +1,9 @@
+import { expect } from 'chai'
+import guessTheNumber from '../src/guessTheNumber'
+
+
+describe('guessTheNumber()', function() {
+  it('should be a function', function() {
+    expect(guessTheNumber).to.be.a('function')
+  })
```

Comment:
Perhaps checking if the result of `guessTheNumber()` is actually a number would be appropriate